File: junghyeok/cheese_front/entity/Share.js
class Share{
    constructor(id, customerId, nickname, branchId, createdAt, sharedPhotoMap){
        this.id = id;
        this.customerId = customerId;
        this.nickname = nickname;
        this.branchId = branchId;
        this.createdAt = createdAt;
        this.sharedPhotoMap = sharedPhotoMap;
    }
}

export default Share;

File: server/Cheese/src/main/java/com/hknu/entity/Share.java
package com.hknu.entity;

import java.sql.Timestamp;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@AllArgsConstructor
public class Share {
	private final Integer share_id;
	private final Integer customer_id;
	private final Integer branch_id;
	private final Timestamp created_at;
}
